README.md

```diff
@@ -34,7 +34,14 @@ This will produce a single plugin file (with support for multiple architectures)
 dist/com.example.my-plugin.tar.gz
 ```
 
-There is a build target to automate deploying and enabling the plugin to your server, but it requires configuration and [http](https://httpie.org/) to be installed:
+There is a build target to automate deploying and enabling the plugin to your server, but it requires configuration of a [personal access token](https://docs.mattermost.com/developer/personal-access-tokens.html):
+```
+export MM_SERVICESETTINGS_SITEURL=http://localhost:8065
+export MM_ADMIN_TOKEN=j44acwd8obn78cdcx7koid4jkr
+make deploy
+```
+
+or login credentials:
```

Comment:
What do you think about reversing the order? Deploying the plugin to a local instance is the usual case and a access token is not as handy, as it gets deleted on a server nuke.